firebase-appdistribution/test-app/src/main/java/com/googletest/firebase/appdistribution/testapp/MainActivity.kt

```diff
@@ -9,7 +10,7 @@ import androidx.appcompat.app.AppCompatActivity
 import androidx.appcompat.widget.AppCompatButton
 import com.google.firebase.appdistribution.AppDistributionRelease
 import com.google.firebase.appdistribution.FirebaseAppDistribution
-import com.google.firebase.appdistribution.FirebaseAppDistributionException
+import com.google.firebase.appdistribution.FirebaseAppDistributionException as FirebaseAppDistributionException1
```

Comment:
Why the `as FirebaseAppDistributionException1`?

Reply:
This was a change rachel made. I wasn't changing anything in testApp and it doesn't impact customers so I decided to leave rachel's wip on the testapp as is.

Reply:
Sounds good!